res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- TODO(Tomasito): Change this to the version in which KeyboardController support will be added-->
+<MixxxKeyboardPreset schemaVersion="1" mixxxVersion="2.0.0+">
+  <info>
+    <name>Keyboard Default</name>
+    <author>Mixxx</author>
+    <description>Default keyboard mapping. (Layout: en_US)</description>
+    <wiki>https://www.mixxx.org/manual/latest/chapters/appendix.html#appendix-keyboard</wiki>
+  </info>
+
+  <controller>
+    <group name="[AutoDJ]">
```

Comment:
What is the reason, not to adopt the midi controller XML schema? 
We could probably all the midi facilities if we do so. 
```
            <control>
                <group>[Channel1]</group>
                <key>LPD8RK.softVolume</key>
                <status>0xB0</status>
                <midino>0x65</midino>
                <options>
                    <script-binding/>
                </options>
          </control>
```